king_phisher/server/database/models.py

```diff
@@ -397,6 +398,95 @@ class Message(Base):
 	credentials = sqlalchemy.orm.relationship('Credential', backref='message', cascade='all, delete-orphan')
 	visits = sqlalchemy.orm.relationship('Visit', backref='message', cascade='all, delete-orphan')
 
+@register_table
+class ModuleQuestion(Base):
```

Comment:
I'd just rename this `TestQuestion` so we don't use "Module" as a prefix as well. I get that Question is under module in the heirarchy but within the context of this entire file, it'd make sense to group it with the other Test stuff.